test/rfqm_test.ts

```diff
@@ -641,6 +642,108 @@ describe(SUITE_NAME, () => {
+                takerToken: contractAddresses.etherToken,
+                pool: '0x',
+                salt: '0',
+                chainId: 1337,
+                verifyingContract: '0xd209925defc99488e3afff1174e48b4fa628302a',
+                txOrigin: MOCK_WORKER_REGISTRY_ADDRESS,
+                expiry: new BigNumber('1903620548'),
+            };
+
+            const params = new URLSearchParams({
+                buyToken: 'ZRX',
+                sellToken: 'WETH',
+                sellAmount: sellAmount.toString(),
+                takerAddress,
+                intentOnFilling: 'false',
+                skipValidation: 'true',
+            });
+
+            const expectedPrice = '1.5';
+            return rfqtMocker.withMockedRfqtQuotes(
+                [
+                    {
+                        // Quote from MM 1
+                        endpoint: MARKET_MAKER_1,
+                        requestApiKey: API_KEY,
+                        requestParams: {
+                            ...BASE_RFQM_REQUEST_PARAMS,
+                            takerAddress,
+                            sellAmountBaseUnits: sellAmount.toString(),
+                            buyTokenAddress: contractAddresses.zrxToken,
+                            sellTokenAddress: contractAddresses.etherToken,
+                        },
+                        responseCode: 200,
+                        responseData: {
+                            signedOrder: {
+                                ...BASE_SIGNED_ORDER,
+                                chainId: 1, // Not the chain we're on
+                                makerAmount: wrongChainQuote,
+                                takerAmount: sellAmount,
+                                signature: {
+                                    ...VALID_SIGNATURE,
+                                },
+                            },
+                        },
+                    },
+                    {
+                        // Quote from MM 2
+                        endpoint: MARKET_MAKER_2,
+                        requestApiKey: API_KEY,
+                        requestParams: {
+                            ...BASE_RFQM_REQUEST_PARAMS,
+                            takerAddress,
+                            sellAmountBaseUnits: sellAmount.toString(),
+                            buyTokenAddress: contractAddresses.zrxToken,
+                            sellTokenAddress: contractAddresses.etherToken,
+                        },
+                        responseCode: 200,
+                        responseData: {
+                            signedOrder: {
+                                ...BASE_SIGNED_ORDER,
+                                makerAmount: correctChainQuote,
+                                takerAmount: sellAmount,
+                                signature: {
+                                    ...VALID_SIGNATURE,
+                                    r: '0xb1',
```

Comment:
I think that `r` and `s` needs to be the same length as a legit signature, otherwise it won't pass the JSON schema check

Reply:
Hmmm... seems to work for me. it's copied from the test above.